src/app/pages/TopicPage/TopicPage.jsx

```diff
@@ -11,10 +14,33 @@ const Wrapper = styled.div`
 `;
 
 const TopicPage = ({ pageData }) => {
+  const { lang } = useContext(ServiceContext);
+  const { title, description, promos } = pageData;
+
+  const promoEntities = promos.map(promo => ({
```

Comment:
is this maybe the type of thing that should be in the BFF?

Reply:
Hmm that would mean returning two sets of promo data from the bff - one for UI, one for Schema.org items

Reply:
This is quite similar to the discussion around iChef url composition. I'd say the BFF has returned all the data the frontend needs and the frontend is composing it into a format suitable for rendering a page. I think this might belong in Simorgh but we are still working this stuff out though.

Reply:
sounds good